src/components/navigation/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import moon from '../../assets/icon-moon.svg';
+import sun from '../../assets/icon-sun.svg';
+import { useTheme } from '../../context/ThemeContext';
+
+export const ThemeSwitcher = () => {
+	const { theme, toggleTheme } = useTheme();
+
+	const handleToggleTheme = () => {
+		if (theme === 'light') {
+			localStorage.setItem('theme', 'dark');
```

Comment:
moglbys to ogarnac jednolinijkowcem: 
`localStorage.setItem('theme', theme === 'light' ? 'dark' : 'light')`

Reply:
done